accern_xyme/accern_xyme.py

```diff
@@ -1298,7 +1298,103 @@ def __repr__(self) -> str:
 # *** NodeHandle ***
 
 
-class CSVBlobHandle:
+EMPTY_BLOB_PREFIX = "null://"
+
+
+class BlobHandle:
+    def __init__(
+            self,
+            client: XYMEClient,
+            uri: str,
+            is_full: bool,
+            pipeline: PipelineHandle) -> None:
+        self._client = client
+        self._uri = uri
+        self._is_full = is_full
+        self._pipeline = pipeline
+
+    def is_full(self) -> bool:
+        return self._is_full
+
+    def is_empty(self) -> bool:
+        return self._uri.startswith(EMPTY_BLOB_PREFIX)
+
+    def get_uri(self) -> str:
+        return self._uri
+
+    def get_pipeline(self) -> PipelineHandle:
+        return self._pipeline
+
+    def get_content(self) -> Optional[ByteResponse]:
+        if not self.is_full():
+            raise ValueError(f"URI must be full: {self}")
+        if self.is_empty():
+            return None
+        fin, ctype = self._client._raw_request_bytes(METHOD_POST, "/uri", {
+            "uri": self._uri,
+            "pipeline": self.get_pipeline().get_id(),
+        })
+        return interpret_ctype(fin, ctype)
+
+    def list_files(self) -> List:
+        if self.is_full():
+            raise ValueError(f"URI must not be full: {self}")
```

Comment:
return the file in a list?

Reply:
if it's a full blob, the function is useless